tools/projmgr/test/data/TestSolution/test.csolution.yml

```diff
@@ -9,6 +9,8 @@ solution:
           C-CPP: [-C-CPP-type-CM0]
           ASM: [-ASM-type-CM0]
           Link: [-Link-type-CM0]
+          Link-C: [-Link-C-type-CM0]
+          Link-CPP: [-Link-CPP-type-CM0]
```

Comment:
Can you add some examples for multiple Linker flags?

Reply:
Does this make sense? Flags under misc share the same API to transfer to cprj format.